packages/web/docs/src/app/page.tsx

```diff
@@ -3,30 +3,52 @@ import {
   Anchor,
   ArchDecoration,
   CallToAction,
+  cn,
   DecorationIsolation,
   GetYourAPIGameRightSection,
   Heading,
   HighlightDecoration,
   LargeHiveIconDecoration,
   ToolsAndLibrariesCards,
 } from '@theguild/components';
-import { cn } from '../lib';
-import { CheckIcon } from './check-icon';
-import { CommunitySection } from './community-section';
-import { AligentLogo, KarrotLogo, LinktreeLogo, MeetupLogo, SoundYXZLogo } from './company-logos';
-import { CompanyTestimonialsSection } from './company-testimonials';
-import { EcosystemManagementSection } from './ecosystem-management';
-import { FeatureTabs } from './feature-tabs';
-import { FrequentlyAskedQuestions } from './frequently-asked-questions';
-import { Hero, HeroFeatures, HeroLinks, TrustedBy } from './hero';
-import { InfoCard } from './info-card';
-import { Page } from './page';
-import { StatsItem, StatsList } from './stats';
-import { TeamSection } from './team-section';
+import { CheckIcon } from '../components/check-icon';
+import { CommunitySection } from '../components/community-section';
+import {
+  AligentLogo,
+  KarrotLogo,
+  LinktreeLogo,
+  MeetupLogo,
+  SoundYXZLogo,
+} from '../components/company-logos';
+import { CompanyTestimonialsSection } from '../components/company-testimonials';
+import { EcosystemManagementSection } from '../components/ecosystem-management';
+import { FeatureTabs } from '../components/feature-tabs';
+import { FrequentlyAskedQuestions } from '../components/frequently-asked-questions';
+import { Hero, HeroFeatures, HeroLinks, TrustedBy } from '../components/hero';
+import { InfoCard } from '../components/info-card';
+import { LandingPageContainer } from '../components/landing-page-container';
+import { StatsItem, StatsList } from '../components/stats';
+import { TeamSection } from '../components/team-section';
+import { metadata as rootMetadata } from './layout';
+
+export const metadata = {
+  title: 'Open-Source GraphQL Federation Platform',
+  description:
+    'Fully Open-Source schema registry, analytics and gateway for GraphQL federation and other GraphQL APIs',
+  alternates: {
+    // to remove leading slash
+    canonical: '.',
```

Comment:
It's so weird

Reply:
no other way to remove leading slash for index page